### DISCUSSION TITLE
When i click "Proccess my application" it proccessing and then do nothing

### DISCUSSION THREAD

**Discussion by MrGhost23** [community/community#70227] [GitHub Education]
### Hi everyone,

I don't know what's the problem but it just does nothing

![Untitled](https://github.com/community/community/assets/72778139/6e607060-b0a5-4384-932c-edc37e79015d)

**Comment by ks1686** [comment]
Running into the same issue, can't figure out how to contact support.

**Comment by OmkarOza** [comment]
You can check your internet connection or try to do same thing in incognito mode I also facing same issue which resolve by this solution.

**Reply by MrGhost23** [reply]
Unfortunately, tried all of that and from many devices still doesn't work.

**Comment by ks1686** [comment]
Found the solution: whatever doc you're submitting is returning an internal error they don't show you on your end. You'll have to either submit a new doc or tweak the one you currently have.

My university's unofficial transcript ended up working for me. First doc I was tryna submit was an instant verification form.

**Reply by MrGhost23** [reply]
Thank you.

**Comment by 51n4k4r1m1** [comment]
same as [here](https://github.com/orgs/community/discussions/70470#discussioncomment-7342956)
